src/main/java/org/truffleruby/language/arguments/KeywordArgumentsDescriptor.java

```diff
@@ -16,14 +16,27 @@
  * arguments were passed. The callee must handle that for now. */
 public class KeywordArgumentsDescriptor extends ArgumentsDescriptor {
 
+    public enum SplatType {
```

Comment:
Maybe should be KeywordSplatType, if imported statically anywhere.
`SplatType` on its own sounds like a `*ary` splat.